awx/main/tasks.py

```diff
@@ -1919,6 +1920,8 @@ def build_private_data(self, inventory_update, **kwargs):
                     group_prefix = v
                 elif k == 'satellite6_want_hostcollections' and isinstance(v, bool):
                     want_hostcollections = v
+                elif k == 'satellite6_want_ssh_ansible_host' and isinstance(v, bool):
```

Comment:
oopsy! should be `satellite6_want_ansible_ssh_host` for our dear docs readers to be able to prognosticate what to put

Extra sharing moment:
---------------------------------
I personally feel very sad when i read docs and it says (my interpretation) "You see a var you want to set in foreman.ini!? Cool, write it here except put `satellite6`, e.g. `satellite6_my_fav_var`"

There is a difference between `satellite6` and `satellite6_` !!!!
Also it is very hard to spell word.
also, what if I am using katello or foreman?
:crying_cat_face:

Reply:
oh my gosh, thanks so much for finding that